pkg/template/api/install/install.go

```diff
@@ -92,22 +92,21 @@ func newRESTMapper(externalVersions []unversioned.GroupVersion) meta.RESTMapper
 	for i := len(externalVersions) - 1; i >= 0; i-- {
 		worstToBestGroupVersions = append(worstToBestGroupVersions, externalVersions[i])
 	}
-
-	rootScoped := sets.NewString("SubjectAccessReview", "SelfSubjectAccessReview")
```

Comment:
@enj this is clearly dev error, iow. copy&paste one. It doesn't make sense to say these resources are root scoped for this api group, we already state that explicitly in their appropriate group. I haven't seen any error coming out of that and it's good this was discovered and fixed.